src/main/java/com/zaxxer/hikari/metrics/prometheus/HikariCPCollector.java

```diff
@@ -18,20 +18,26 @@
 
 import com.zaxxer.hikari.metrics.PoolStats;
 import io.prometheus.client.Collector;
+import io.prometheus.client.CollectorRegistry;
 import io.prometheus.client.GaugeMetricFamily;
+
 import java.util.Arrays;
 import java.util.Collections;
+import java.util.HashSet;
 import java.util.List;
 import java.util.Map;
+import java.util.Set;
 import java.util.concurrent.ConcurrentHashMap;
 import java.util.function.Function;
 
-class HikariCPCollector extends Collector {
+public class HikariCPCollector extends Collector {
```

Comment:
Why should ``HikariCPCollector`` be ``public``?  This is considered an internal class.

Reply:
If we use only one collector and provides custom CollectorRegistry, why not make users are able to use this HikariCPCollector as a default implemention? So that users can use this or write another implemention extends HikariCPCollector.

Reply:
@brettwooldridge